sonar-db/src/main/java/org/sonar/db/permission/template/PermissionTemplateMapper.java

```diff
@@ -70,9 +70,9 @@
 
   int countUserLoginsByQueryAndTemplate(@Param("query") PermissionQuery query, @Param("templateId") long templateId);
 
-  List<String> selectGroupNamesByQueryAndTemplate(@Param("query") PermissionQuery query, @Param("templateId") long templateId, RowBounds rowBounds);
+  List<String> selectGroupNamesByQueryAndTemplate(@Param("organizationUuid")String organizationUuid, @Param("templateId") long templateId, @Param("query") PermissionQuery query, RowBounds rowBounds);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Split this 199 characters long line (which is greater than 180 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS00103)